beacon-chain/p2p/service.go

```diff
@@ -203,6 +204,17 @@ func (s *Service) connectWithAllPeers(multiAddrs []ma.Multiaddr) {
 	}
 }
 
+func logIP4Addr(id peer.ID, addrs ...ma.Multiaddr) {
+	var correctAddr ma.Multiaddr
+	for _, addr := range addrs {
+		if strings.Contains(addr.String(), "/ip4/") {
+			correctAddr = addr
+			break
+		}
+	}
+	log.Infof("Node's listening multiaddr is %s", correctAddr.String()+"/p2p/"+id.String())
```

Comment:
the fix should be good for copy and paste as i saw earlier .
did you check it is working using this printout?

Reply:
yep , I tried it out